Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -118,6 +118,14 @@
         "BadgeUpdateVersionRequired": "/Assets/Badges/windows10_anniversary.png"
       },
       {
+        "Name": "Loading",
+        "Type": "LoadingPage",
+        "About": "Loading control helps to show any xaml content with animation to the user while the app is doing some calculation.",
```

Comment:
"LoadingControl helps to show content with animation to the user while the app is doing some calculation."